main.js

```diff
@@ -34,7 +34,7 @@ if (process.argv.includes('--clearCacheDir')) {
   rimraf.sync(app.getPath('userData'));
 }
 
-app.disableHardwareAcceleration();
+// app.disableHardwareAcceleration();
```

Comment:
is it required?

Reply:
Yes.  WebGL does not work with hardware acceleration off.  I will remove this rather than commenting out before this is merged.